### ISSUE TITLE
Add name to user

### ISSUE THREAD

**Issue by paltorg** [mathieudutour/slacking-pigeons#2]
Great stuff, we have been looking for something like this. The other chats are way to bulky. Currently, you only see the image from Slack when responding, would it be possible in the future to add a name? This would make it more "personal".

**Comment by mathieudutour** [comment]
Yeah I was thinking about showing the name when you hover on the avatar. What do you think?

**Comment by paltorg** [comment]
Actually I would prefer just having the name next to the avatar. Keeping it simple! 👍

**Comment by Braunson** [comment]
I'd prefer to be able to pass an object of userdata for the currently logged in user. Right now you could set it via labels but that's not ideal, especially if you wanted to set the avatar.
